R/tags.R

```diff
@@ -679,17 +690,21 @@ takeHeads <- function(ui) {
 #'
 #' @param tags A tag-like object to search for dependencies.
 #' @param tagify Whether to tagify the input before searching for dependencies.
+#' @inheritParams resolveDependencies
 #'
 #' @return A list of \code{\link{htmlDependency}} objects.
 #'
 #' @export
-findDependencies <- function(tags, tagify = TRUE) {
+findDependencies <- function(tags, tagify = TRUE, resolveTagFunction = TRUE) {
```

Comment:
When would `resolveTagFunction` be FALSE? Would it make sense to export `resolveFunctionalDependencies` instead?

Reply:
Good point, until we find some reason to not to resolve, I'm on-board with always resolving and exporting `resolveFunctionalDependencies()`